modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CreateZoneCommand.java

```diff
@@ -136,7 +142,8 @@ private CatalogZoneDescriptor descriptor(int objectId) {
                 ),
                 Objects.requireNonNullElse(dataNodesAutoAdjustScaleDown, INFINITE_TIMER_VALUE),
                 Objects.requireNonNullElse(filter, DEFAULT_FILTER),
-                fromParams(storageProfileParams)
+                fromParams(storageProfileParams),
+                Objects.requireNonNullElse(consistencyMode, STRONG_CONSISTENCY)
```

Comment:
Shouldn't we validate that `consistencyMode` has only appropriate values and throw appropriate CatalogValidationException with detailed message otherwise?